mysql-test/suite/perfschema/r/show_sanity.result

```diff
@@ -672,6 +674,10 @@ where (show_mode = "JUNK: GLOBAL-ONLY")
 and source = "I_S.SESSION_STATUS")
 order by show_mode, source, variable_name;
 SHOW_MODE	SOURCE	VARIABLE_NAME
+5.7	P_S.SESSION_STATUS	Innodb_encryption_n_merge_blocks_decrypted
```

Comment:
Was show_sanity updated correctly? I.e. no need to mention the new vars in the testcase itself anywhere?